bedrock/products/templatetags/misc.py

```diff
@@ -155,7 +155,7 @@ def vpn_total_price(ctx, plan="12-month", country_code=None, lang=None):
 
 @library.global_function
 @jinja2.contextfunction
-def vpn_saving(ctx, plan="12-month", country_code=None, lang=None):
+def vpn_saving(ctx, plan="12-month", country_code=None, lang=None, ftl_string="vpn-shared-pricing-save-percent", ftl_files=FTL_FILES):
```

Comment:
See comment above: suggest adding the new string to `products/vpn/shared.ftl`, so you can avoid needing to pass `ftl_files` around.
```suggestion
def vpn_saving(ctx, plan="12-month", country_code=None, lang=None, ftl_string="vpn-shared-pricing-save-percent"):
```